pom.xml

```diff
@@ -71,6 +59,67 @@
                     <target>8</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
what about this, why you need it?

Reply:
to prepare the project to be published on jpm